### PR TITLE
(MAINT) added ticket process doc & meta docs folder

### PR DESCRIPTION
**Pull request by kevpl**

### CODE REVIEW

File: docs/meta/ticket_process.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Pre-Development States
+
+These are the states that a Beaker ticket goes through before it is picked up for
+development. Note that this is only for work done by the Beaker team. If there
+is something you'd like to contribute to, we would appreciate that you go
+through the development states for tracking purposes, but you can skip directly
+to the "In Progress" state if you're going to work on a ticket.
+
+- Open
```
Comment (tvpartytonight) [inline docs/meta/ticket_process.md]: The way this formats looks kind of weird; the bullet becomes somewhat indented, but the text is still left-aligned; maybe use the `###` instead?

**Comment by kevpl** [comment]
@puppetlabs-jenkins retest this please

File: docs/meta/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The meta folder contains docs not about how to **use** Beaker, but how to
+deal with Beaker in other ways, such as how to **change** it.
+
+**Table of Contents**
+
+- [Ticket Process](docs/meta/ticket_process.md)
```
Comment (tvpartytonight) [inline docs/meta/README.md]: I think that this link is broken? Should it just point to `ticket_process.md`?

Comment (kevpl) [inline docs/meta/README.md]: thanks for catching that @tvpartytonight! I'd meant to do a quick validate after pushing and before submitting the PR, but spaced on it. Links should be good now

**Comment by kevpl** [comment]
the failures are unrelated to the change, & I'm disinclined to keep shooting for green, giving the scope of this change.

**Comment by tvpartytonight** [comment]
👍

**Pull request merged**